package.json

```diff
@@ -24,6 +22,7 @@
     "material-ui-password-field": "1.4.3",
     "md5": "2.2.1",
     "moment": "2.22.2",
+    "npm": "6.3.0",
```

Comment:
Should should not be added. Make changes in travis config if you wish to upgrade npm.